hisensetv/__init__.py

```diff
@@ -226,6 +239,60 @@ def send_key(self, keyname: str):
         """
         self._call_service(service="remote_service", action="sendkey", payload=keyname)
 
+    def launch_app(self, app: str):
```

Comment:
Can you add a docstring here, or prefix with `_` to indicate that this is an internal function?